website/docs/about/contribution.md

```diff
@@ -1,34 +1,110 @@
+
+To get an idea of what this process could look like, please refer to [the process followed by the Design System team when building a net-new component](https://github.com/hashicorp/design-system/blob/main/packages/components/NEW-COMPONENT-CHECKLIST.md).
+
+### Implementation and release
+
+Once reviewed and approved, the feature can be implemented and released in Helios. 
+
+- The Design System team will:
+    - Document these artifacts on the Helios website
+    - Release the artifacts that were created/ported/updated in collaboration with the product team
+    - Publicize the release of these new artifacts, as well as the fact that they were the result of of a joint collaboration with the product team
+- The product team will:
+    - Follow the guidance provided by the Design System team in the follow-up work related to the collaboration effort
+    - Have a clear plan to adopt the artifacts produced during the collaboration
+
+## Direct contributions
+
+We support a small number of autonomous but coordinated direct contributions. These include bug fixes in the code, new icons, documentation improvements, and small enhancements to exitsting components.
+
+### Contribute a new icon in Figma
+
+If you’d like to add a new icon to the [icon library](/icons/library), follow the steps in the [Figma library README](https://www.figma.com/file/MYiw4kiVpunIMMw0sBkE1t/%E2%9C%8F%EF%B8%8F-Flight-Development?node-id=566%3A1129&t=Bbflj3UUaWVyhamn-4).
 
-While we don’t accept direct contributions of new components or patterns, we do welcome contributions in the following forms:
+### Contribute in code
 
-* [Reporting](https://go.hashi.co/hds-support) or fixing an issue or bug (see [Code contributions](#code-contributions))
-* [Providing general feedback](https://docs.google.com/forms/d/e/1FAIpQLSdPShbSiUYgimppZeqWT99bVi6zLbKDThQUQTirH3RCj2okdg/viewform)
-* [Proposing new components or patterns](https://go.hashi.co/hds-support)
-* Adding an icon (see [Icon contributions](#icon-contributions))
-* Improving our documentation
-* Participating in discussions or coming to Office Hours
+To work in one of the Helios code packages, consult the corresponding `CONTRIBUTING.md` file for:
+- [Tokens](https://github.com/hashicorp/design-system/blob/main/packages/tokens/CONTRIBUTING.md)
+- [Ember Flight Icons](https://github.com/hashicorp/design-system/blob/main/packages/ember-flight-icons/CONTRIBUTING.md)
+- [Components](https://github.com/hashicorp/design-system/blob/main/packages/components/CONTRIBUTING.md)
 
-We continue to listen to feedback as we consider the best way to open contributions further.
+Review the [Engineering Checklist for new components](https://github.com/hashicorp/design-system/blob/main/packages/components/NEW-COMPONENT-CHECKLIST.md#engineering-checklist) if you’re working on a net-new component.
```

Comment:
```suggestion
Review the [Engineering Checklist for new components](https://github.com/hashicorp/design-system/blob/main/packages/components/NEW-COMPONENT-CHECKLIST.md#engineering-checklist) if you’re working on a new component.
```

Reply:
Btw, to me just saying "new" sounds simpler and is clear enough so I've been editing the references here but if you prefer to keep the "net new" term then just remove the dash.